codegen-on-oss/examples/context_server_example.py

```
@@ -0,0 +1,115 @@
+    commit = sys.argv[2] if len(sys.argv) > 2 else None
+
+    # Server URL
+    server_url = "http://localhost:8000"
+
+    print(f"Using CodeContextRetrievalServer at {server_url}")
+    print(f"Repository: {repo_full_name}")
+    print(f"Commit: {commit or 'latest'}")
+
+    # Check if server is running
+    try:
+        response = requests.get(server_url)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)